collect_stata/write_json.py

```diff
@@ -0,0 +1,369 @@
+"""write_json.py"""
+__author__ = "Marius Pahl"
+
+import json
+from collections import Counter, OrderedDict
+
+import numpy as np
+import pandas as pd
+
+
+def sorting_dataframe(values, labels, missings, frequencies):
+    """Method to sort values and labels and return sorted dict"""
```

Comment:
This is a function.
It sorts four lists by the contents of one list.

Reply:
See #18